server.go

```diff
@@ -357,16 +357,18 @@ func (t *ServerClient) ServerPut(putArgs shared.PutArgs, reply *shared.Clock) er
 	if exists == false {
 		// Create a new entry into the inFlightDb
 		inFlightDb[key] = newValue
+		*reply = newValue.Ts
 	} else {
 		// Compare the timeStamps of the values and either update or ignore
 		ordering := util.TotalOrderOfEvents(prevValue.Ts, prevValue.ServerId, newValue.Ts, newValue.ServerId)
 		if ordering == util.HAPPENED_BEFORE {
 			inFlightDb[key] = newValue
+			*reply = newValue.Ts
+		} else {
+			*reply = prevValue.Ts
```

Comment:
Not sure if we should return prevValue.Ts or newValue.Ts in this case. Probably, it is fine.